src/Microsoft.AspNetCore.Mvc.Abstractions/ModelBinding/Metadata/IModelBindingMessageProvider.cs

```diff
@@ -25,6 +25,13 @@ public interface IModelBindingMessageProvider
         Func<string> MissingKeyOrValueAccessor { get; }
 
         /// <summary>
+        /// Error message the model binding system adds when no value is provided for the request body,
+        /// but a value is required.
+        /// </summary>
+        /// <value>Default <see cref="string"/> is "A value for the request body was not provided.".</value>
+        Func<string> MissingRequestBodyRequiredValueAccessor { get; }
```

Comment:
@SteveSandersonMS wasn't too upset with adding the property but wanted to hear from @Eilon on the breaking change. So, go ahead but you'll need to add an `exceptions.netcore.json` file with the right content to keep API Check happy.